R/fourier_transform.R

```diff
@@ -1,6 +1,6 @@
-library(testthat)
-library(ggplot2)
-library(dplyr)
+import(testthat)
+import(gglot2)
+import(dplyr)
```

Comment:
import is not a function in R. I think you might be confusing this with python! 
In R, these statements should not be here at the top of the file at all. That's why you have the dependency listed in the DESCRIPTION file and you use the double colon notation to specify what package the function is coming from. You can just just delete these lines.

Reply:
oh I see! Thanks for pointing it out!